docs/src/pinn/system.md

```diff
@@ -363,3 +363,207 @@ eqs  = @. [(Dxx(u_(x,y)) + Dyy(u_(x,y))) for u_ in u] ~ -sin(pi*x)*sin(pi*y)*[0
 # Initial and boundary conditions
 bcs = [u[1](x,0) ~ x, u[2](x,0) ~ 2, u[3](x,0) ~ 3, u[4](x,0) ~ 4]
 ```
+
+
+## Linear parabolic system of PDEs
+
+We can use NeuralPDE to solve the linear parabolic system of PDEs described here http://eqworld.ipmnet.ru/en/solutions/syspde/spde0101.pdf.
```

Comment:
it will be nice if you add a latex formula, like in other examples.
![image](https://user-images.githubusercontent.com/12683885/125798866-7fe59560-d201-4055-b891-1a892f758803.png)

Reply:
Note that documenter will render native Tex so you don't need to do the image thing